rasa/shared/utils/common.py

```diff
@@ -5,32 +5,59 @@
 import logging
 from typing import Text, Dict, Optional, Any, List, Callable, Collection
 
+import rasa.shared.utils.io as io_utils
```

Comment:
please use the complete module path (python code convention)

Reply:
gosh I don't like this convention :D